Unable to compile due to failure to find freeimage

### System Information

7d30fc65af0d81610c6af1f913d67986e91c4f62 on Fedora 36 6.0.5-200.fc36.x86_64

### Expected Behavior

-  running CMake without any error

### Steps to Reproduce

- clone repository recursively
- either
  A) Open the trenchbroom in QtCreator and attempt to configure it
  B) Run trenchbroom with cmake-gui and attempt to configure it
  C) Run `cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH="cmake/packages"` in build directory

```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find tinyxml2 (missing: tinyxml2_LIBRARY tinyxml2_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/packages/tinyxml2-config.cmake:28 (find_package_handle_standard_args)
  vcpkg/scripts/buildsystems/vcpkg.cmake:834 (_find_package)
  CMakeLists.txt:138 (find_package)
  ```

--- a/Build.md
+++ b/Build.md
@@ -97,12 +97,6 @@ If you have a debian-based distribution, open a command prompt and execute this 
 sudo apt-get install g++ qt5-default libqt5svg5-dev libxi-dev libgl1-mesa-dev libglu1-mesa-dev freeglut3-dev mesa-common-dev libglew-dev libxrandr-dev build-essential libglm-dev libxxf86vm-dev libfreetype6-dev libfreeimage-dev libtinyxml2-dev pandoc cmake p7zip-full ninja-build curl
 ```
 
-Or, on Fedora:
-
-```bash
-sudo dnf install g++ cmake qt5-qtbase-devel qt5-qtsvg-devel ninja-build pandoc mesa-libGLU-devel
-```
-
 ### Build TrenchBroom
 
 Create a subdirectory in TrenchBroom directory called `build`.

